.github/workflows/rust.yml

```
@@ -60,14 +60,22 @@ jobs:
   test:
     needs: diff
     if: needs.diff.outputs.isRust == 'true'
-    runs-on: [self-hosted, self-hosted-ubuntu]
+    runs-on: ${{ matrix.os }}
     strategy:
+      matrix:
+        os: [[self-hosted, self-hosted-ubuntu], windows-latest]
```

```suggestion
        os:
          - [self-hosted, self-hosted-ubuntu]
          - windows-latest
```

This might be a bit more readable

---

(assuming it works the same way)

---

Yep, sounds good